core/src/main/scala/org/apache/spark/rdd/DoubleRDDFunctions.scala

```diff
@@ -155,7 +155,7 @@ class DoubleRDDFunctions(self: RDD[Double]) extends Logging with Serializable {
    * to the right except for the last which is closed
    *  e.g. for the array
    *  [1, 10, 20, 50] the buckets are [1, 10) [10, 20) [20, 50]
-   *  e.g 1<=x<10 , 10<=x<20, 20<=x<=50
+   *  e.g 1&lt;=x&lt;10 , 10&lt;=x&lt;20, 20&lt;=x&lt;=50
```

Comment:
note to myself, It seems inlined tags such as 

```
{@code ... < ...}
```

and

```
{@literal >}
```

work also okay for both but they are valid ones for javadoc. For scaladoc, they are dealt with monospace text (like ` `<` ` or `` `... < ...` ``). As genjavadoc seems not replacing it, it seems they work apparently. I guess we should avoid those though.